### ISSUE TITLE
To use backdrop-filter on iOS and Safari on macOS it must be prefixed

### ISSUE THREAD

**Issue by Grawl** [logux/logux.io#7]
For now, we have this:

```css
@supports (backdrop-filter: blur(15px)) {
    @media (width < 1024px) {
        .layout_aside {
            background: transparent;
            backdrop-filter: blur(15px);
        }
    }
}
```

But `backdrop-filter` works on Mobile Safari and Safari on macOS, it should be prefixed:

```css
@supports ((-webkit-backdrop-filter: blur(15px)) or (backdrop-filter: blur(15px))) {
    @media (width < 1024px) {
        .layout_aside {
            background: transparent;
            -webkit-backdrop-filter: blur(15px);
                    backdrop-filter: blur(15px);
        }
    }
}
```

**Comment by ai** [comment]
Seems like we have an Autoprefixer issue here

**Comment by ai** [comment]
Fixed 3f29fd583c79197591ebe60847c678608eeeccd4

**Comment by Grawl** [comment]
oh 🙈
